src/DependencyInjection/HappyrAuth0Extension.php

```diff
@@ -50,6 +50,13 @@ public function load(array $configs, ContainerBuilder $container)
 
         $configProviderDefinition = $container->getDefinition(ConfigurationProvider::class);
 
+        // if extra keys
+        if (isset($configs[1]) && [] !== $configs[1]) {
```

Comment:
Why $configs[1] ?

Reply:
This is the first time Im using the "extra keys feature". 

So `$configs` is an array of config values. Those are processed in `$this->processConfiguration()` into a `$config` array. 

The `$configs[1]` will always include the "extra keys". (if my debugging is correct)

Reply:
I don't think it work like that.
If i remember correctly, the $configs array contain all `happyr_auth0` nodes if confirgured in multiples files.
You need to use only the result of `$this->processConfiguration()` which is in chage to aggregate all of them.
The result contains declared keys alongside extrax keys.

Reply:
Oh. you might be correct. I'll do some better debugging. 

Thank you